pkg/machines/components/networks/networkOverviewTab.jsx

```diff
@@ -0,0 +1,171 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2019 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { networkId } from '../../helpers.js';
+import { WarningAlert } from '../notification/inlineNotification.jsx';
+import cockpit from 'cockpit';
+
+import './networkOverviewTab.css';
+import 'form-layout.less';
+
+const _ = cockpit.gettext;
+
+const DHCPHost = (host, index, family, idPrefix) => {
+    const id = `${idPrefix}-${family}-dhcp-host-${index}`;
+
+    let hostVals = [];
+    if (host.name)
+        hostVals.push(_("Name: ") + host.name);
+    if (host.mac) // MAC for ipv4, ID for ipv6
+        hostVals.push("MAC: " + host.mac);
+    else if (host.id)
+        hostVals.push("ID: " + host.id);
+    if (host.ip)
+        hostVals.push("IP: " + host.ip);
+
+    const hostInfo = hostVals.join(", ");
+
+    return (<React.Fragment key={index}>
+        <label className='control-label' htmlFor={id}> {`DHCP host ${index + 1}:`} </label>
+        <div id={id}> {hostInfo} </div>
+    </React.Fragment>);
+};
+
+export const NetworkOverviewTab = ({ network, actionError, actionErrorDetail, onActionErrorDismiss }) => {
+    const idPrefix = `${networkId(network.name, network.connectionName)}`;
+
+    let qosInboundVals = [];
+    if (network.bandwidth.inbound && network.bandwidth.inbound.average)
+        qosInboundVals.push(_("Average: ") + network.bandwidth.inbound.average);
```

Comment:
I couldn't find example of inline details in cockpit, so if you would want it withou colons, I'm fine with that.

Reply:
So Average is a bad example because it's sublabel. But autostart, persistent, outbount, address, netmask etc should be all be shown in consistent way.

Reply:
Yes I agree, i forgot about those colons there. 
But should colons be left in inline details (bandwidth and HTCP host)?

Reply:
@andreasn should be the right person to ask since he did the mockup for this feature.

Reply:
His mockup contained colons, but yeah if it could interfere translation then lets discuss it :)

Reply:
How does it interfere with translations exactly? Is it specifically the ":" character that is the problem? If so, you can either show it as `&#58;` in html or `content: "\003A"; ` with css.
https://www.toptal.com/designers/htmlarrows/punctuation/colon/